examples/kubernetes/namespace-template.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: {{namespace}}
+  labels:
+    name: {{namespace}}
```

Comment:
I've seen it in examples but I don't think I use the label anyway so I deleted it.